src/components/common/YourCompany/YourCompany.vue

```diff
@@ -226,7 +209,7 @@
 
     <!-- Business Start Date (changes, conversions and firm corrections only -->
     <template v-if="isChangeRegFiling || isFirmConversionFiling || isFirmCorrectionFiling">
-      <section class="section-container">
+      <section class="section-container business-start-date">
```

Comment:
Since there should only be 1 business-start-date, I recommend using id instead of class.

Reply:
I agree.